dev-packages/e2e-tests/test-applications/create-remix-app-v2/tests/behaviour-server.test.ts

```diff
@@ -10,7 +10,7 @@ test('Sends two linked transactions (server & client) to Sentry', async ({ page
   const httpServerTransactionPromise = waitForTransaction('create-remix-app-v2', transactionEvent => {
     return (
       transactionEvent.type === 'transaction' &&
-      transactionEvent.contexts?.trace?.op === 'http.server' &&
+      transactionEvent.contexts?.trace?.op === 'http' &&
```

Comment:
hmm, we should fix this to be `http.server` - this is important I believe!

Reply:
Marked `requestHandler` spans as `http.server` 👍